package.json

```diff
@@ -14,7 +14,7 @@
     "body-parser": "latest",
     "eslint": "2.8.x",
     "ejs": "2.5.x",
-    "request": "2.74.x"
+    "request": "2.87.x"
```

Comment:
Could you also update the overall package version number?

Reply:
Done and merged 👍